Forms/HTMLEditor/FileShortcodeProvider.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace SilverStripe\Forms\HtmlEditor;
```

Comment:
I feel slightly uncomfortable, moving the providers to separate namespaces, as the embed provider would stay in the Form\HtmlEditor?
I've for now moved them all into `SilverStripe\View\Parsers`. Welcome any thoughts? Happy to move them around, but I'd personally prefer to keep all providers in the same namespace, so CMS Providers (like sitetree_link), can go into `SilverStripe\CMS\Chosen\Namespace` as well, to keep it all in a unified place.